src/includes/getting-started-primer/react-native.mdx

```diff
@@ -0,0 +1,20 @@
+<Note>
+
+_Sentry's React Native SDK enables automatic reporting of errors and exceptions, and identifies perfomance issues in your application. The React Native SDK includes performance monitoring (our `@sentry/tracing` package) by default._
+
+</Note>
+
+Features:
+
+- Automatic Native Crash [Error Tracking](/product/error-monitoring/) (using both `sentry-cocoa` & `sentry-android`)
+- Offline storage of events
+- Events enriched with device data
```

Comment:
Link to https://sentry-docs-git-update-react-native-content.sentry.dev/platforms/react-native/enriching-events/context/ maybe?

Reply:
Done. Linked to /platforms/react-native/enriching-events/context/. Thanks.